x/src/test.rs

```diff
@@ -17,16 +23,66 @@ pub struct Args {
     #[structopt(long, short)]
     /// Only run unit tests
     unit: bool,
+
+    #[structopt(long)]
+    /// Do not fast fail the run if tests (or test executables) fail.
```

Comment:
This gets printed out as help output, so it'll be good to have a standard. Generally I've seen really short sentences without periods at the end.

```suggestion
    /// Do not fast fail the run if tests (or test executables) fail
```

Reply:
I'll do my best to remember this, and ensure I keep 'em so.    Cleaned up (for now).